unit_test/sparse/Test_Sparse_spmv_blockcrs.hpp

```diff
@@ -145,7 +145,7 @@ void check_blockcrs_times_v(const char fOp[], scalar_t alpha, scalar_t beta,
       for (lno_t ib = 0; ib < blockSize; ++ib) {
         const lno_t my_row     = ir * blockSize + ib;
         mat_rowmap[my_row + 1] = mat_rowmap[my_row] + (jend - jbeg) * blockSize;
-        for (lno_t ijk = jbeg; ijk < jend; ++ijk) {
+        for (auto ijk = jbeg; ijk < jend; ++ijk) {
```

Comment:
I believe this should be `size_type` since jbeg and jend are values coming from the row_map.